scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -19,38 +19,34 @@ This quickstart assumes that you are familiar with Terra workspaces, the data mo
     - BigQuery data editor
     - BigQuery job user
     - BigQuery Read Session User
-4. These tools expect reblocked gvcf files as input. 
+4. These tools expect re-blocked gVCF files as input.
 
 ## 1. Import Data
 In order to load data into BigQuery without hitting daily load limits, we recommend you group your input files into samples sets and follow these steps for each sample set.
 
-## 1.1 Assign Gvs Ids 
-To optimize the internal queries, each sample must have a unique and consecutive integer id assigned. Run the `GvsAssignIds` workflow. This will create an appropiate id for each sample in the sample set and update the BigQuery data set with the sample name to id mapping info.
+A sample set for the quickstart has already been created with 10 samples and paths to re-blocked gVCFs for each sample.  Run the two import workflows against this sample set by selecting "sample_set" as the root entity type ("Step 1") and `gvs_demo-10` for the data ("Step 2").  If you are creating your own sample set, note that the sample table should have a column for the re-blocked gVCFs (`hg38_reblocked_gvcf` or `reblocked_gvcf_path`) and their index files need to be in the same location.
+
+## 1.1 Assign Gvs IDs
+To optimize the internal queries, each sample must have a unique and consecutive integer ID assigned. Run the `GvsAssignIds` workflow, which will create an appropriate ID for each sample in the sample set and update the BigQuery dataset with the sample name to ID mapping info.
 
 These are the required parameters which must be supplied to the workflow:
 
 | Parameter      | Description |
 | ----------------- | ----------- |
 | project_id | The name of the google project containing the dataset |
 | dataset_name      | The name of the dataset you created above       |
-| external\_sample_names | Use `this.samples.sample_id` |
```

Comment:
Isn't this a required parameter to the workflow?

Reply:
It is, but when the user clones and then runs the workflow, the saved inputs should be populated automatically.  I thought that was how it was set up before, so I was going along with that method, but I'd also be happy to put that in.